app/.env

```diff
@@ -0,0 +1,2 @@
+WEATHER_API_KEY=
```

Comment:
Please move these to config.py.example. The default should be `os.getenv('WEATHER_API_KEY')`.

Reply:
While trying to push my commits, I've encountered a problem: All checks have failed
I think it is probably because there is no API KEY for the tests. :-(
How will the actual KEY be set?
The status of the push is error due to that...

Reply:
Read about API Mocking to prevent using the API (which might cost money, or have rate limit) every time we run the tests.

It will also solve this problem :)

Reply:
Thanks!

Reply:
Thanks!